### PR TITLE
added cli command to init shuttle

### PR DESCRIPTION
**Pull request by franklovefrank**
- adds CLI command "shuttle" to initialize shuttle 
- updated contributing documentation for clarity and formatting consistency

### CODE REVIEW

**Comment by franklovefrank** [comment]
don't have permission to link issue but this should resolve 
https://github.com/application-research/estuary/issues/685

File: main.go [hunk 1]
```diff
@@ -589,6 +589,32 @@ func main() {
 				}
 				return cfg.Save(configFile)
 			},
+		}, {
+			Name:  "shuttle",
```
Comment (en0ma) [inline main.go:593]: change this to match with the api `shuttle-init`

File: main.go [hunk 2]
```diff
@@ -589,6 +589,32 @@ func main() {
...
+			Usage: "Initializes a shuttle node, returns handle and authorization token",
+			Action: func(cctx *cli.Context) error {
+				configFile := cctx.String("config")
+				if err := cfg.Load(configFile); err != nil && err != config.ErrNotInitialized { // still want to report parsing errors
+					return err
+				}
+				shuttle := &model.Shuttle{
```
Comment (en0ma) [inline main.go:600]: move this model to the where it gets created in the db

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,28 +70,31 @@ ____________________________________O/_______
...
-$ curl -H "Authorization: Bearer ESTb43c2f9c-9832-498a-8300-35d9c4b8c16eARY" -X POST localhost:3004/admin/shuttle/init
-{"handle":"SHUTTLE4e8b1770-326c-4c95-9976-7cc1ee12244bHANDLE","token":"SECRET7528ab25-1266-4fa4-86cf-719a43bbcb4fSECRET"}
+$ curl -H "Authorization: Bearer REPLACE_ME_WITH_API_KEY" -X POST localhost:3004/admin/shuttle/
+{"handle":"<your_handle_printed_here>", "token":"<your_auth_token_printed_here>"}
 ```
 
-Using the output from the above command, start a shuttle node in development mode
+or by using the CLI 
+```bash 
+$ ./estuary shuttle
```
Comment (en0ma) [inline CONTRIBUTING.md:85]: after you makes the changes, here becomes `shuttle-init`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -72,28 +70,31 @@ ____________________________________O/_______
...
 
 Build the shuttle binary
-
 ```bash
 $ make estuary-shuttle
 ```
 
-Then, initialize a shuttle node using the Estuary API endpoint `/admin/shuttle/init` or the admin UI at <host>/admin/shuttle
-
+Then, initialize a shuttle node. You can do this by using the admin UI at `<host>/admin/shuttle`, by posting to the Estuary API endpoint `/admin/shuttle/init`
```
Comment (en0ma) [inline CONTRIBUTING.md:77]: nit: `, or by posting to the Estuary API en` - you missed "or"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -72,28 +70,31 @@ ____________________________________O/_______
...
 $ make estuary-shuttle
 ```
 
-Then, initialize a shuttle node using the Estuary API endpoint `/admin/shuttle/init` or the admin UI at <host>/admin/shuttle
-
+Then, initialize a shuttle node. You can do this by using the admin UI at `<host>/admin/shuttle`, by posting to the Estuary API endpoint `/admin/shuttle/init` 
 ```bash
-$ curl -H "Authorization: Bearer ESTb43c2f9c-9832-498a-8300-35d9c4b8c16eARY" -X POST localhost:3004/admin/shuttle/init
-{"handle":"SHUTTLE4e8b1770-326c-4c95-9976-7cc1ee12244bHANDLE","token":"SECRET7528ab25-1266-4fa4-86cf-719a43bbcb4fSECRET"}
+$ curl -H "Authorization: Bearer REPLACE_ME_WITH_API_KEY" -X POST localhost:3004/admin/shuttle/
```
Comment (en0ma) [inline CONTRIBUTING.md:79]: you missed `init` - `-X POST localhost:3004/admin/shuttle/init`

File: CONTRIBUTING.md (hunk 4 above)
Comment (franklovefrank) [reply CONTRIBUTING.md:77]: it's a list of 3 with or at the end :)

File: main.go (hunk 2 above)
Comment (franklovefrank) [reply main.go:600]: think i understood

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -72,28 +70,30 @@ ____________________________________O/_______
...
 ```
 
-Using the output from the above command, start a shuttle node in development mode
+or by using the CLI 
+```bash 
+$ ./estuary shuttle-init
+{"handle":"<your_handle_printed_here>", "token":"<your_auth_token_printed_here>"}
+```
 
+Using the ouxtput from the above command, start a shuttle node in development mode:
```
Comment (en0ma) [inline CONTRIBUTING.md]: nit: `ouxtput`

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -72,28 +70,30 @@ ____________________________________O/_______
...
 ```bash
-$ ./estuary-shuttle --dev --estuary-api=localhost:3004 --auth-token=SECRET7528ab25-1266-4fa4-86cf-719a43bbcb4fSECRET --handle=SHUTTLE4e8b1770-326c-4c95-9976-7cc1ee12244bHANDLE --logging --host=localhost:3005
+$ ./estuary-shuttle --dev --estuary-api=localhost:3004 --auth-token=SECRETfd17213b-48ad-4937-8d70-3946855920a2SECRET --handle=SHUTTLEd55cd9ee-7ad7-4978-b7f0-2b36f0fdfeffHANDLE --logging --host=localhost:3005
 ```
 output
-
 ```
 Wallet address is:  <your_estuary-shuttle_address_printed_here>
-2021-09-16T14:47:54.353-0700    INFO    dt-impl impl/impl.go:145        start data-transfer module
+2021-09-16T14:47:54.353-0700    INFO    dt-impl impl/impl.xo:145        start data-transfer module
```
Comment (en0ma) [inline CONTRIBUTING.md]: nit: `dt-impl impl/impl.xo:145`

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -72,28 +70,30 @@ ____________________________________O/_______
...
+or by using the CLI 
+```bash 
+$ ./estuary shuttle-init
+{"handle":"<your_handle_printed_here>", "token":"<your_auth_token_printed_here>"}
+```
 
+Using the ouxtput from the above command, start a shuttle node in development mode: 
 ```bash
-$ ./estuary-shuttle --dev --estuary-api=localhost:3004 --auth-token=SECRET7528ab25-1266-4fa4-86cf-719a43bbcb4fSECRET --handle=SHUTTLE4e8b1770-326c-4c95-9976-7cc1ee12244bHANDLE --logging --host=localhost:3005
+$ ./estuary-shuttle --dev --estuary-api=localhost:3004 --auth-token=SECRETfd17213b-48ad-4937-8d70-3946855920a2SECRET --handle=SHUTTLEd55cd9ee-7ad7-4978-b7f0-2b36f0fdfeffHANDLE --logging --host=localhost:3005
```
Comment (en0ma) [inline CONTRIBUTING.md]: can we replace these secrets with <place_holders>

**Review by en0ma — APPROVED** [review]

**Pull request merged**